dm/tests/dmctl_command/run.sh

```diff
@@ -142,6 +145,39 @@ function run() {
 	fi
 }
 
+function test_full_mode_conn() {
```

Comment:
please check the actual connection is same as you expected by SHOW PROCESSLIST

Reply:
Did you mean I should `start-task` and see if they are as expected?

Reply:
Yes. if you do this check, you can find the forgotten checkpoint/DDL connections on your own.